app/preprints/template.hbs

```diff
@@ -1,9 +1,18 @@
 {{page-title this.theme.provider.providerTitle replace=true}}
-<div {{with-branding (get-model this.theme.provider.brand)}}>
-    <BrandedNavbar
-        @translateKey='collections.general.brand'
-        @brandRoute='preprints.discover'
-        @addLinkKey='collections.navbar.add'
-    />
-    {{outlet}}
-</div>
+{{#if this.theme.isProvider}}
+    <div {{with-branding (get-model this.theme.provider.brand)}}>
+        <ThemeStyles />
+        <BrandedNavbar
+            @translateKey='collections.general.brand'
+            @brandRoute='preprints.discover'
+            @addLinkKey='collections.navbar.add'
```

Comment:
I know this is not code that you have added, but why `collections.general.brand` and `collections.navbar.add`?